### ISSUE TITLE
 The `add-path` command is deprecated and will be disabled on November 16th.

### ISSUE THREAD

**Issue by aminya** [actions/setup-node#212]
There is a deprecation in setup-node. 

Error: The `add-path` command is deprecated and will be disabled on November 16th. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/

Edit: 

⚠️ The main branch of this repository includes a high-severity bug, and it should not be used. ⚠️ 
For setup-node only a patch has been applied to v1 and v2-beta versions of GitHub actions (not the main branch). 

Apparently, Google gave GitHub 90 days to fix the problem. Read the full story: 

> Google to GitHub: Time's up – this unfixed 'high-severity' security bug affects developers
> https://www.zdnet.com/article/google-to-github-times-up-this-unfixed-high-severity-security-bug-affects-developers/

This is the same for my branch with X86 support (unfortunately I made that PR against main branch): #194.

**Comment by peaceiris** [comment]
Why don't you use the latest version? The latest version is v1.4.4

**Comment by aminya** [comment]
I am using my branch which supports x86 arch: https://github.com/aminya/setup-node

It is only 2 commits behind. I will update the release. Hopefully, they merge #194 faster so I can use the official version.

**Comment by thboop** [comment]
Can you provide your failing workflow @aminya ?

The latest version of v1 and v2-beta should be working, we patched them. Here's the workflow I used to test:
```
name: testnode
on: [push]
jobs:
  test:
    strategy:
      matrix:
        os: ['windows-latest', 'ubuntu-latest', 'macos-latest']
    runs-on: ${{matrix.os}}
    steps:
      - uses: actions/setup-node@v2-beta
      - uses: actions/setup-node@v1
```

**Comment by aminya** [comment]
@thboop I am using the main branch (#194 which is based on main). I can't use the latest release because it does not support x86 architecture.

**Comment by NomanGul** [comment]
updating the version number from `actions/setup-node@v1.4.2` to `actions/setup-node@v1.4.4` fixed the issue for me :+1:

**Comment by staabm** [comment]
we are running actions/setup-node@v2.1.0 which also contains the reported problem

**Comment by aminya** [comment]
I updated the description of the issue to include the back story of this high-severity bug. People should avoid the main branch of this repository.

**Comment by aminya** [comment]
This is fixed now in v2.1.3

**Comment by dracos** [comment]
setup-node@v1 has just started giving this error.

**Comment by SumiyaE** [comment]
thanx

**Comment by milleniait** [comment]
> This is fixed now in `actions/setup-node@v2.1.3`

Join the All New Cross chain project Dogelana $DGLN.

Doge and Solana introduces the best and fastest Network on Blockchain.
Join us to partake in our IEO/ICO/NFT GIVEAWAY

https://t.me/+ZtYF6PzehrMxYmE0
